components/search/TierToggler.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <p class="font-italic">
+      {{ toggleText }}
+      <nuxt-link
+        :to="$route.fullPath"
+        class="toggle-button"
+        data-qa="tier toggle button"
+        @click.native="toggleHandler"
+      >
+        {{ !active ? button.show : button.hide }}
+      </nuxt-link>.
+    </p>
+  </div>
+</template>
+
+<script>
+  export default {
+    name: 'TierToggler',
+
+    props: {
+      activeState: {
+        type: Boolean,
+        default: null
+      }
+    },
+
+    data () {
+      return {
+        active: false,
+        button: {
+          show: this.$t('searchTier.button.show'),
+          hide: this.$t('searchTier.button.hide')
+        },
+        text: {
+          show: this.$t('searchTier.text.show'),
+          hide: this.$t('searchTier.text.hide')
+        }
+      };
+    },
+
+    computed: {
+      toggleText () {
+        if (this.active) {
+          return this.text.hide;
+        }
+        return this.text.show;
+      }
+    },
+
+    mounted () {
+      if (this.activeState) {
+        this.active = this.activeState;
+      }
+    },
+
+    methods: {
+      toggleHandler () {
+        if (!this.active) {
+          this.$emit('click', 'contentTier', ['*']);
```

Comment:
click is quite generic. I suggest either, making this consistent with how facets are applied and emitting 'changed', or emitting 'toggle'.
The function being picked up on in pages/search/index.vue will need to be updated accordingly.

Reply:
Made this 'toggle'